cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -722,6 +722,8 @@
       public static final String PROGRAM = "prg";
       public static final String PROGRAM_TYPE = "prt";
       public static final String PROGRAM_ENTITY = "ent";
+      public static final String PLUGIN_NAME = "plg";
```

Comment:
Do we have to use these short-names for all new tags as well? Maybe we can start migrating to better names for new tags?

Reply:
yeah unfortunately this is an artifact of the CDAP metrics system. There are short names defined here, and corresponding long names in MetricsQueryHelper.java. Metrics queries use the long names.